### PR TITLE
Model training, dataset creation and heatmaps visualization code

### PR DESCRIPTION
**Pull request by codingsneha**

### CODE REVIEW

File: create_merged_data.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Create pickle dump of dictionary of numpy arrays rangeResult(n,182,256), velocity(n,), L_R(n,)
```
Comment (arghasen10) [inline create_merged_data.py]: remove these.

File: find_peaks_velocity.py
Comment (arghasen10) [inline find_peaks_velocity.py:1]: Don't we need this file anymore?

File: helper.py [hunk 2]
```diff
@@ -554,78 +529,8 @@ def get_mode_velocity(velocity_array_framewise):
...
-        y.append(velocity)
-    print(X)
-    # Convert lists to numpy arrays
-    X = np.asarray(X, dtype=object)
-    y = np.array(y)
-    
-    # Splitting data into train and test sets
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
-    
-    return X_train, X_test, y_train, y_test
```
Comment (arghasen10) [inline helper.py:631]: So I guess you have passed all these codes in test_helper script?

File: vicon_estimate.py [hunk 3]
```diff
@@ -157,22 +157,50 @@ def get_gt_velocity(filename):
...
+    # plt.figure(figsize=(15,3))
+    # df_vel.groupby('sec')['vel'].mean().plot(label='mean')
+    # df_vel.groupby('sec')['vel'].median().plot(label='median')
+    # plt.ylabel('Velocity (cm/sec)')
+    # plt.legend()
+    # plt.ylabel('Vicon Estimated Speed')
+    # plt.xlabel('No. of Frames')
+    # plt.savefig('vel_vicon.pdf')
+    # plt.show()
     return df_vel.groupby('sec')['vel'].mean()
```
Comment (arghasen10) [inline vicon_estimate.py]: Keep the plots; no need to comment them out.

**Review by arghasen10 — CHANGES_REQUESTED** [review]
Please check the comments.

File: find_peaks_velocity.py
Comment (codingsneha) [reply find_peaks_velocity.py:1]: No sir it is redundant, everything is included in create_merged_data.py

File: create_merged_data.py (hunk 1 above)
Comment (codingsneha) [reply create_merged_data.py]: Done

File: helper.py (hunk 2 above)
Comment (codingsneha) [reply helper.py:631]: Yes sir

File: vicon_estimate.py (hunk 3 above)
Comment (codingsneha) [reply vicon_estimate.py]: Done

**Review by arghasen10 — APPROVED** [review]
LGTM.

**Pull request merged**